packages/components/src/components/heading/heading.tsx

```diff
@@ -0,0 +1,63 @@
+import { Component, Element, h, Host, JSX, Prop } from '@stencil/core';
+import {
+  AllowedTypes,
+  attachComponentCss,
+  getDataThemeDarkAttribute,
+  THEMES,
+  TEXT_ALIGNS,
+  validateProps,
+} from '../../utils';
+import type { PropTypes, TextAlign, TextColor, Theme } from '../../types';
+import type { HeadingTag, HeadingVariant } from './heading-utils';
+import { getHeadingTagName, HEADING_TAGS } from './heading-utils';
+import { getComponentCss } from './heading-styles';
+
+const propTypes: Omit<PropTypes<typeof Heading>, 'variant'> = {
+  // variant: AllowedTypes.string, // TODO: with all the different values this can't easily be validated
```

Comment:
Shouldn't this be cleaned up / reduced with the new component? Currently it seems just like a rename.